docs/pywbemcli/generaloptions.rst

```diff
@@ -513,11 +513,25 @@ bypasses client side verification of the WBEM server certificate.
 ``--timestats`` general option
 """"""""""""""""""""""""""""""
 
-The ``--timestats`` general option is a boolean option that enables the
-gathering and display of time
-statistics on the interactions with the WBEM server.  If enabled, the
-time statistics are output after each command is executed including the
-operations executed, the size of the operations, and the execution time.
+The ``--timestats`` / ``-T`` general option is a choice option that enables the
+gathering and display of time statistics on the interactions with the WBEM
+server.
+
+This option allows enabling statistics in either of 2 modes:
+
+1. 'auto' - a statistics report is displayed after each command is executed in
+the interactive mode and after the defined command is executed in the command
+mode.
+
+2. 'cmd' - Statistics are gathered but the report is displayed only upon
+execution of the ``statistics show`` command. Note that the ``statistics reset``
+command can be used to reset the statistics counters.
+
+There is a third choice, 'none' that can be used in the interactive mode of 
+pywbemcli to disable time statistics.
```

Comment:
Simplification: remove none so that the way the user turns off display in interactive mode is to set  timestats cmd

Reply:
DONE

Reply:
This one depends on the outcome of the discussion on the other comment about none being the default.

Reply:
I believe that the difference between "cmd" and "none" is not visible to the user (other than the increased complexity to have both), and suggest to remove the "none" choice. This means that "cmd" becomes the default. That also allows in interactive mode to turn off an inherited auto mode.

Reply:
See discussion in earlier command.  where one of the proposals is to return the command to boolean but do it with --timestats/--no-timestats which would just set the equivalent of the auto mode since we would always have stats gathering enabled.  The command would become just a display option.

I left the other item with **DISCUSSION** set.

Reply:
DONE. Modifed per our discussions.  Note that this change is a separate git commit

Reply:
I think this is a good decision.